main.nf

```diff
@@ -98,6 +96,33 @@ Channel
 
 
 /*
+ * PREPROCESSING - Build Bowtie index for mature and hairpin
+ */
+process makeBowtieIndex {
+
+    publishDir "${params.outdir}/bowtie/reference", mode: 'copy'
```

Comment:
It would be nice to not save this by default, only when `--saveReference` is specified. See the NGI-RNAseq pipeline for an example of this:

https://github.com/SciLifeLab/NGI-RNAseq/blob/master/main.nf#L188-L189
```groovy
publishDir path: { params.saveReference ? "${params.outdir}/reference_genome" : params.outdir },
           saveAs: { params.saveReference ? it : null }, mode: 'copy'
```